server/tests/test_server_pages.py

```diff
@@ -7,7 +7,7 @@
 
 
 @pytest.mark.parametrize("logged_in", [True, False])
-def test_index_view(client, two_test_notebooks, fake_user, logged_in):
+def test_index_view(client, ten_test_notebooks, fake_user, logged_in):
```

Comment:
I think we should probably just inline the ten_test_notebooks fixture, since there's a lot of logic about it that's pretty specific to this test (e.g. that only every other notebook has 10 revisions) and the behavior we want to test will almost certainly change, perhaps soon. The fixtures in conftest.py are meant to be broadly reusable across a large range of tests.